packages/select/src/components/FzSelectOption.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <button
+    @click="$emit('click')"
```

Comment:
```suggestion
    @click="emit('click')"
```
this way you're using the return value of `defineEmits`